### PR TITLE
chore: (platform) update Travis to run platform tests

### PR DESCRIPTION
**Pull request by Betrozov**
#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/2868

#### Please provide a brief summary of this pull request.
Travis will be run platform tests not only core tests

#### Please check whether the PR fulfills the following requirements

- [X] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [X] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [0.20.1-rc.15](https://github.com/SAP/fundamental-ngx/compare/v0.20.1-rc.14...v0.20.1-rc.15) (2020-07-16)
```
Comment (droshev) [inline CHANGELOG.md:5]: that should not be part of this PR

**Comment by droshev** [comment]
> Again you must be loving avatar ;-) that its part of the PR that shoudl only deal with travis .. I think here we should see only travis changed and maybe scripting part under package.json

@Betrozov I think you need to rebase your PR or something

**Comment by valorkin** [comment]
> > Again you must be loving avatar ;-) that its part of the PR that shoudl only deal with travis .. I think here we should see only travis changed and maybe scripting part under package.json
> 
> @Betrozov I think you need to rebase your PR or something

I usually update fork master before review and then push update PR to latest button :D
